### PR TITLE
tạo spec cho model favorite_pitch

### PR DESCRIPTION
**Pull request by vinh-nd2002**
## Related Tickets
- [ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/69557)

## WHAT (optional)
- Viết spec cho model FavoritePitch

## HOW
- Tạo file spec/models/favorite_pitches_spec.rb

## Evidence (Screenshot or Video)
**Rubocop**
rubocop
**Spec**
Screenshot from 2023-10-03 22-14-51

### CODE REVIEW

**Comment by thainda-0252** [comment]
ok

**Comment by vinh-nd2002** [comment]
ready

File: spec/models/favorite_pitches_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+RSpec.describe FavoritePitch, type: :model do
+  it "is valid with valid attributes" do
+    user = create(:user)
+    football_pitch = create(:football_pitch)
+    favorite_pitch = create(:favorite_pitch)
+    expect(favorite_pitch).to be_valid
+  end
+
+  it "requires a user" do
```
Comment (ngocvt-0484) [inline spec/models/favorite_pitches_spec.rb]: sao e k dùng các validate hỗ trợ sẵn như
```
 RSpec.describe Booking, type: :model do
     context "associations" do
         it {is_expected.to belong_to :user}
     end
     context "validates" do
         it {is_expected.to validate_presence_of :booking_from}
     end
 end
```

**Comment by vinh-nd2002** [comment]
ready

**Pull request merged**